src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestValidationParameters.cs

```diff
@@ -109,6 +109,20 @@ public class SignedHttpRequestValidationParameters
         /// <remarks>https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-5.1</remarks>
         public bool AcceptUnsignedHeaders { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the claims to validate if present.
+        /// </summary>
+        /// <remarks>
+        /// Validation will only occur if <see cref="ValidatePresentClaims"/> is set to <c>true</c>.
+        /// </remarks>
+        public IEnumerable<string> ClaimsToValidateWhenPresent { get; set; } = new List<string>
+        {
+            SignedHttpRequestClaimTypes.Ts,
```

Comment:
Should we remove 'Ts' and 'U' from the collection? 
'M' and 'P' were mentioned as recommended claims.

Reply:
Good question. I was defaulting recommended and required claims. 
We would expect the usual case is Ts and U are part of the always validated claims.

Reply:
I agree. The reason for this feature is using 'M' and 'P' as recommended claims, but we are providing additional extensibility so that any claim could be validated in this fashion.

Having only 'M' and 'P' in this collection would reduce number of steps that users have to do to enable this scenario. In this case users would flip ValidateM and ValidateP switches to `false` and ValidatePresentClaims to `true`.

Reply:
This is a good point. I'll make the change